src/sass/contents/ContentsFilter.scss

```diff
@@ -107,7 +107,7 @@
     }
 
     &--unpublished {
-      background: $color-unpublish;
```

Comment:
will the $color-unpublish stay red for other stuff?

Reply:
red will not be used in current flow anywhere. As, red is indicating that something was "published" and then someone "unpublished" it. Which, I discussed with Joe, Filipe and Eugenio and in current version we dont have support for it and we are ignoring it. So, now we have only three colors. Grey, Yellow and Green.